jenkins/model_output_processing_Jenkinsfile

```diff
@@ -71,6 +72,21 @@ pipeline {
           echo ${DATE} > date.txt
           aws s3 cp date.txt s3://prod-owi-resources/resources/Application/wbeep/${TIER}/date.txt
         '''
+        script {
+          def files = findFiles(glob: 'order_of_magnitude_test.txt')
+          if (files) {
+            if (files[0].length > 200) {
+              mail to: 'mhines@usgs.gov, wwatkins@usgs.gov, lplatt@usgs.gov',
+                   subject: "Suspicious data: ${currentBuild.fullDisplayName}",
+                   body: "Pipeline finished, but had suspicious data. See -- ${files[0]} -- in the job workspace for details."
```

Comment:
Not sure if we want to deal with it now, but I'm not sure this file will be available to look at, since it is being created on the slave node.  It seems like only the master workspace gets saved right now.  This might be useful: https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#code-stash-code-stash-some-files-to-be-used-later-in-the-build

Reply:
So far, I've been able to go find the file in the build. If not, at least it has done the job of notifying about weird data and we can always go run the code [here](https://github.com/usgs-makerspace/wbeep-processing/pull/139/files#diff-240ffcd594d4186bbb11e8b7a30bb3a9R50) manually to figure out problem HRUs.